Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/server/BaseService.java

```diff
@@ -201,7 +202,7 @@ private static void readHttpHeaders(MessageContext messageContext, AssertionType
         Message message = ((WrappedMessageContext) messageContext).getWrappedMessage();
         if (message != null && message.get(Message.PROTOCOL_HEADERS) != null) {
```

Comment:
If message has isEmpty or isNotEmpty method , use that, instead of if (message != null && message.get(Message.PROTOCOL_HEADERS) != null)